HANDBOOK.md

```diff
@@ -42,11 +42,11 @@
   
 ***
 ### 4. Setting up
-* Grant yourself the "ctf_map_editor" priv by running `/grantme ctf_map_editor`. You can also use `/grantme all` to grant you "ctf_map_editor" and all the other privs that will be useful while making a map, such as fly, noclip, fast, etc...
-* You can hit Esc on your keyboard and click `Change Keys` to see your controls (such as how to fly, noclip, etc...).
+* Grant yourself the "ctf_map_editor" priv by running `/grantme ctf_map_editor`. You can also use `/grantme all` to grant yourself "ctf_map_editor" and all the other privs that will be useful while making a map, such as fly, noclip, fast, etc.
+* You can hit <kbd>Esc</kbd> on your keyboard and click `Change Keys` to see your controls (such as how to fly, noclip, etc.).
 
   ![image](https://github.com/CTF-handbooks/map-maker-handbook/assets/88883098/98117763-d42c-4a64-ade6-a894c14f61a9)
-* *Tip*: run `/time 12000` and `/set -n time_speed 0` if you want to disable nighttime while building the map.
+* *Tip*: Run `/time 12000` and `/set -n time_speed 0` if you want to disable nighttime while building the map.
```

Comment:
```suggestion
* *Tip*: run `/time 12000` and `/set -n time_speed 0` if you want to disable nighttime while building the map.
```
I don't think upper case is needed here since "Tips" starts with an upper case letter.